pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1347,7 +1403,7 @@ func handleAppNetworkModify(ctxArg interface{}, key string,
 	publishAppNetworkStatus(ctx, status)
 
 	if !config.Activate && status.Activated {
-		doInactivateAppNetwork(ctx, status)
+		doInactivateAppNetwork(ctx, status, false)
```

Comment:
In don't understand by you need to set deleteFlag=false in this case. What is key is that the Allocate and Free calls line up and currently they do not, in particular if there are any errors.